openwisp_controller/pki/base/models.py

```diff
@@ -22,6 +30,12 @@ class AbstractCert(ShareableOrgMixin, BaseCert):
 
     class Meta(BaseCert.Meta):
         abstract = True
+        constraints = [
+            models.UniqueConstraint(
+                fields=['common_name', 'organization'],
+                name='%(app_label)s_%(class)s_comman_name_and_organization_is_unique',
+            ),
```

Comment:
I guess we'll have trouble upgrading instances which have certificates with duplicated common names, which can happen. 
We would need to run a data migration before doing this, we could loop over all certs and look if there's another one named the same way and if so regenerate it with the new naming.

I guess the same could happen for the CA although the probabilities are far slimmer.